### DISCUSSION TITLE
Documentation regarding how katana works / katana components

### DISCUSSION THREAD

**Discussion by zlw9991** [projectdiscovery/katana#1123] [Q&A]
May I ask if there are any specific resources regarding how katana works, on say at least an abstract level? Such as what tools or libraries are at play when katana does webcrawling & how they interact with each other to give a json output? I'm planning to use it in a project & would like to draw up a diagram of how it works. Sorry for any inconvenience caused.

**Comment by GeorginaReeder** [comment]
Hey @zlw9991 ! Great question - for resources, most of what we have available is in our docs here: https://docs.projectdiscovery.io/tools/katana/overview and the readme that's available in GitHub: https://github.com/projectdiscovery/katana/blob/main/README.md

If that doesn't contain what you're looking for, I'll tag @dwisiswant0 for other possible resources.
